### DISCUSSION TITLE
Add Websocket support on hardhat config

### DISCUSSION THREAD

**Discussion by 0xask3** [PaulRBerg/hardhat-template#140] [Ideas]
Would it be possible to add WebSocket URL support for chain config in hardhat? As I understand, hardhat doesn't have a standard interface for WebSocket connections but would be great if we could use it for deployments and custom scripts

**Comment by PaulRBerg** [comment]
WebSocket URLs don't work out of the box with Hardhat. I remember having issues at deployment.

So I'm almost certainly not going to modify the template to use WSS instead of HTTPS. But feel free to fork this and tailor the template to your needs.
